pom.xml

```diff
@@ -2232,6 +2236,19 @@
         <version>${version.org.w3c.css.sac}</version>
       </dependency>
 
+      <!-- libraries for incremental compilation-->
+      <dependency>
+        <groupId>commons-cli</groupId>
+        <artifactId>commons-cli</artifactId>
+        <version>${version.commons.cli}</version>
+      </dependency>
+
+      <dependency>
+        <groupId>com.google.collections</groupId>
+        <artifactId>google-collections</artifactId>
+        <version>${version.google.collections}</version>
+      </dependency>
```

Comment:
Looks like the latest version of `google-collections` is almost 8 years old. Do we want to add this kind of dependency? Which module requires this?